app/gimlet/rev-b.toml

```diff
@@ -0,0 +1,551 @@
+bus = "front"
+address = 0x49
+device = "tmp117"
+name = "South"
+description = "South temperature sensor"
+sensors = { temperature = 1 }
+removable = true
+refdes = "J195"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x4a
+device = "tmp117"
+name = "Southeast"
+description = "Southeast temperature sensor"
+sensors = { temperature = 1 }
+removable = true
+refdes = "J196"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x70
+device = "pca9545"
+description = "U.2 ABCD mux"
+refdes = "U336"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x71
+device = "pca9545"
+description = "U.2 EFGH mux"
+refdes = "U339"
+
+[[config.i2c.devices]]
+bus = "front"
+address = 0x72
+device = "pca9545"
+description = "U.2 IJ/FRUID mux"
+refdes = "U337"
+
+[[config.i2c.devices]]
+bus = "m2"
+address = 0x73
+device = "pca9545"
+description = "M.2 mux"
+refdes = "U422"
+
+[[config.i2c.devices]]
+bus = "m2"
+mux = 1
+segment = 3
+address = 0x4c
+device = "tmp451"
+sensors = { temperature = 1 }
+description = "T6 temperature sensor"
+refdes = "U491"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x24
+device = "tps546b24a"
+description = "A2 3.3V rail"
+pmbus = { rails = [ "V3P3_SP_A2" ] }
+sensors = { temperature = 1, voltage = 1, current = 1 }
+refdes = "U522"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x25
+device = "tps546b24a"
+description = "A0 3.3V rail"
+pmbus = { rails = [ "V3P3_SYS_A0" ] }
+sensors = { temperature = 1, voltage = 1, current = 1 }
+refdes = "U560"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x27
+device = "tps546b24a"
+description = "A2 5V rail"
+pmbus = { rails = [ "V5_SYS_A2" ] }
+sensors = { temperature = 1, voltage = 1, current = 1 }
+refdes = "U524"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x29
+device = "tps546b24a"
+description = "A2 1.8V rail"
+pmbus = { rails = [ "V1P8_SYS_A2" ] }
+sensors = { temperature = 1, voltage = 1, current = 1 }
+refdes = "U561"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x3a
+device = "max5970"
+description = "M.2 hot plug controller"
+refdes = "U275"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x4c
+device = "sbtsi"
+name = "CPU"
+description = "CPU temperature sensor"
+sensors = { temperature = 1 }
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x58
+device = "idt8a34003"
+description = "Clock generator"
+refdes = "U446"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x5a
+device = "raa229618"
+description = "CPU power controller"
+pmbus = { rails = [ "VDD_VCORE", "VDD_MEM_ABCD" ] }
+sensors = { temperature = 2, power = 2, voltage = 2, current = 2 }
+refdes = "U350"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x5b
+device = "raa229618"
+description = "SoC power controller"
+pmbus = { rails = [ "VDDCR_SOC", "VDD_MEM_EFGH" ] }
+sensors = { temperature = 2, power = 2, voltage = 2, current = 2 }
+refdes = "U351"
+
+[[config.i2c.devices]]
+bus = "mid"
+address = 0x5c
+device = "isl68224"
+description = "DIMM/SP3 1.8V power controller"
```

Comment:
Mind adding that it's SP3 1.8V A0 power, given that we have an A1 1.8V?

Reply:
Done